packages/itinerary-body/src/ItineraryBody/place-row.tsx

```diff
@@ -89,12 +116,12 @@ export default function PlaceRow({
               for an interline place
             */}
         {interline && <S.InterlineDot>&bull;</S.InterlineDot>}
-        <S.PlaceName>
+        <S.PlaceName aria-hidden>
           <PlaceName config={config} interline={interline} place={place} />
         </S.PlaceName>
       </S.PlaceHeader>
 
-      <S.TimeColumn>
+      <S.TimeColumn aria-hidden>
         {/* Custom rendering of the departure/arrival time of the specified leg. */}
         <TimeColumnContent isDestination={isDestination} leg={leg} />
```

Comment:
This will hide transit real-time updates or any custom content provided by the `TimeColumnContent`.